docs/user-guide.md

```diff
@@ -59,9 +61,26 @@ helm install akri akri-helm-charts/akri
         # Configure Akri to use MicroK8s' CRI socket
         export AKRI_HELM_CRICTL_CONFIGURATION="--set agent.host.crictl=/usr/local/bin/crictl --set agent.host.dockerShimSock=/var/snap/microk8s/common/run/containerd.sock"
         ```
+        If you don't have a existing kubectl and helm installations, you can add aliases. If you do not want to set an
+        alias, add microk8s in front of all kubectl and helm commands.
+        ```sh
+        alias kubectl='microk8s kubectl'
+        alias helm='microk8s helm3'
+        ```
     1. If using **Kubernetes**, Helm and crictl do not require additional configuration.
 
-1. Install Akri Helm chart and enable the desired Configuration (in this case, ONVIF is enabled). See the [ONVIF Configuration documentation](./onvif-sample.md) to learn how to customize the Configuration. Instructions on deploying the udev Configuration can be found in [this document](./udev-sample.md).
+1. Install Akri Helm chart and enable the desired Configuration (in this case, ONVIF is enabled). See the [ONVIF
+   Configuration documentation](./onvif-sample.md) to learn how to customize the Configuration. Instructions on
+   deploying the udev Configuration can be found in [this document](./udev-sample.md). 
+
+    By default the Controller can be deployed to any control plane or worker node. These settings can be changed by
+    adding extra settings when installing Akri below. If you don't want the Controller to ever be scheduled to control
+    plane nodes, add `--set controller.allowOnControlPlane=false` to your install command below. Conversely, if you only
+    want the Controller to run on control plane nodes, add `--set controller.onlyOnControlPlane=true`. This will
+    guarantee the Controller only runs on nodes with the label (key, value) of (`node-role.kubernetes.io/master`, ""),
+    which is the default label for the control plane node for Kubernetes. However, control plane nodes on MicroK8s and
+    K3s do not have this label by default, so you can add it by running `kubectl label node ${HOSTNAME,,}
```

Comment:
Should we mention here that they can also use the nodeSelectors value to add any existing labels (as K3s has one)?